### PR TITLE
Added welcome bot

### LINKED ISSUE #45: Integrate welcome bot
**Screenshots**
I would like to add welcome bot for first issue, first pull request, etc.
Kindly assign it to me under gssoc. Thank you

### PR DESCRIPTION
**Pull request by fomalhauting**
## Related Issue 

- Info about the related issue 

Closes: #45 

Added a welcome bot

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Additional Info (optional)
Any additional information you want to give

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by adarshvulli — APPROVED** [review]
good to go

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
```
Comment (avinal) [inline .github/config.yml]:
```suggestion
  Hello there!👋 Welcome to the Pixelvibe project!🚀⚡
```
A particular name looks much nice and presentable.

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  
+  Thank you and congrats🎉 for opening your very first issue in this project.
+  Please make sure not to start working on the issue, unless you get assigned to it.😄
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
```
Comment (avinal) [inline .github/config.yml]:
```suggestion
  Hello there!👋 Welcome to the Pixelvibe project!💖
```

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  We will get back to you as soon as we can 😄.
+  
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+firstPRMergeComment: >
+  Hello there!👋
+  
+  Congrats🎉 on getting your first pull request merged! All the best for your amazing open source journey ahead 🚀.
```
Comment (avinal) [inline .github/config.yml]:
```suggestion
  Congrats🎉 on getting your first pull request merged! All the best for your amazing open-source journey ahead 🚀.
  
```
Add a blank line at the end of the file.

**Review by avinal — CHANGES_REQUESTED** [review]
Please do the changes

**Comment by avinal** [comment]
@ankitapuri, please add the app to this repository, we might see it working before merging. https://github.com/behaviorbot/welcome#usage

**Comment by fomalhauting** [comment]
> @ankitapuri, please add the app to this repository, we might see it working before merging. https://github.com/behaviorbot/welcome#usage

I have added it.

**Review by adarshvulli — APPROVED** [review]

**Review by avinal — APPROVED** [review]

**Pull request merged**